api/src/functionalTest/java/uk/gov/hmcts/payment/functional/CMCCardPaymentFunctionalTest.java

```diff
@@ -61,6 +66,10 @@
     private static String SERVICE_TOKEN;
     private static boolean TOKENS_INITIALIZED = false;
 
+    private static final Logger LOG = LoggerFactory.getLogger(CMCCardPaymentFunctionalTest.class);
+
+    private static DateTimeZone zoneUTC = DateTimeZone.UTC;
```

Comment:
ZoneUTC variable is unused. Please check.

Reply:
Fixed it